### DISCUSSION TITLE
VS Code autocomplete?

### DISCUSSION THREAD

**Discussion by Meettya** [unocss/unocss#87] [Q&A] [answered]
Its brilliant tool for styles, thanks and good luck with project!

I try it with attributify and its looks clean and clear for me, but it will be perfect to have autocomplete, as windi at vitesse setup.
Is it possible? Is it at roadmap? May be I can try to little help with it?

PS. I always try to get off css because it looks broken, but with uno it seems less broken, may be near "ok, fine, i do it" :)

**Comment by subhasishdas159** [comment]
Trying out solid js with unocss. Ran into this problem. Vscode doesn't show autocomplete.

**Comment by antfu** [comment, marked as answer]
We have made the support about 3 months ago

- https://github.com/unocss/unocss/pull/723

You should have it with the latest extension and latest unocss installed locally. If you run into bugs please open issues with reproduction. Thanks.
